Lib/test/test_cmd_line.py

```diff
@@ -972,10 +976,19 @@ def test_python_legacy_windows_fs_encoding(self):
 
     @unittest.skipUnless(support.MS_WINDOWS, 'Test only applicable on Windows')
     def test_python_legacy_windows_stdio(self):
-        code = "import sys; print(sys.stdin.encoding, sys.stdout.encoding)"
-        expected = 'cp'
-        rc, out, err = assert_python_ok('-c', code, PYTHONLEGACYWINDOWSSTDIO='1')
-        self.assertIn(expected.encode(), out)
+        def get_stderr_class(legacy_windows_stdio):
+            code = 'import sys; print(type(sys.stderr.buffer.raw))'
+            env = {'PYTHONLEGACYWINDOWSSTDIO': str(int(legacy_windows_stdio))}
+            # use stderr=None as legacy_windows_stdio doesn't affect pipes
+            p = spawn_python('-c', code, env=env, stderr=None)
```

Comment:
GitHub Action would run test with pipe, not with console.
In such case, stderr=None is still pipe.

Adding `creationflags=CREATE_NEW_CONSOLE` would allocate new console for subprocess.

Reply:
creationflags=CREATE_NEW_CONSOLE didn't fix test on GitHub Action...